parking_permits_app/resolvers.py

```diff
@@ -29,6 +31,43 @@
 ]
 
 
+@query.field("getPermits")
+@convert_kwargs_to_snake_case
+def resolve_customer_permits(obj, info, customer_id):
+    try:
+        permits = ParkingPermit.objects.filter(customer__pk=customer_id)
+        payload = {
+            "success": True,
+            "permits": [serialize_permit(permit) for permit in permits],
+        }
+    except AttributeError:  # todo not found
+        payload = {"success": False, "errors": ["Permits item matching {id} not found"]}
+    return payload
+
+
+def serialize_permit(permit):
+    price = permit.parking_zone.get_current_price()
+    vehicle = permit.vehicle
+    contract_type = permit.contract_type
+    return snake_to_camel_dict(
+        {
+            "id": permit.pk,
+            **model_to_dict(permit),
+            "price": {
+                "original": price,
+                "offer": price * decimal.Decimal(0.5),
```

Comment:
Use backend pricing engine to get the offer value here.

Reply:
Done